### PR TITLE
Fix AzDev GetCommitAsync and replace AzDevBuildId to BAR BuildId

### PR DESCRIPTION
**Pull request by lwintjen**

### CODE REVIEW

File: src/Microsoft.DotNet.Darc/src/DarcLib.AzDev/AzureDevOpsClient.cs [hunk 1]
```diff
@@ -662,18 +662,11 @@ private async Task<Commit> GetCommitAsync(string accountName, string projectName
                     HttpMethod.Get,
                     accountName,
                     projectName,
-                    $"_apis/git/repositories/{repoName}/commits",
+                    $"_apis/git/repositories/{repoName}/commits/{sha}?api-version=6.0",
```
Comment (riarenas) [inline src/Microsoft.DotNet.Darc/src/DarcLib.AzDev/AzureDevOpsClient.cs:665]: The version shouldn't be included in the request URI like this. This method takes an optional parameter for which version of the API to use.

File: src/Maestro/Maestro.Web/Api/v2020_02_20/Controllers/BuildsController.cs [hunk 2]
```diff
@@ -182,6 +182,14 @@ public async Task<IActionResult> GetCommit(int buildId)
                 return NotFound();
             }
 
+            build = new Data.Models.Build();
```
Comment (mmitche) [inline src/Maestro/Maestro.Web/Api/v2020_02_20/Controllers/BuildsController.cs]: Intentional?

Comment (riarenas) [reply src/Maestro/Maestro.Web/Api/v2020_02_20/Controllers/BuildsController.cs]: I think you just need a refresh to look at the latest iteration. I was about the make the same comment on that whole block as well.

Comment (lwintjen) [reply src/Maestro/Maestro.Web/Api/v2020_02_20/Controllers/BuildsController.cs]: Yup, shouldn't have commit it in the first place. I removed it afterwards :)

**Review by riarenas — APPROVED** [review]

**Review by mmitche — APPROVED** [review]

**Pull request merged**